queries/c/refactoring.scm

```diff
@@ -0,0 +1,24 @@
+;; Grabs all the local variable declarations. This is useful for scope
+;; variable passing.  Which variables do we need to pass to the extracted
+;; function?
+(declaration
+  declarator: (init_declarator
+    declarator: (_) @definition.local_name
+    value: (_) @definition.local_value)) @definition.local_declaration
+
+;; grabs all the arguments that are passed into the function. Needed for
+;; function extraction, 106.
+((parameter_list
+    (parameter_declaration
+        declarator: (_) @definition.function_argument)))
+
+(function_definition) @definition.block
+(compound_statement) @definition.block
+
+(expression_statement) @definition.statement
```

Comment:
@pranavrao145 Ahhh sorry I should have reviewed this before I just merged the inline node PR. I moved statements to inline Nodes in Treesitter (see typescript or golang treesitter files). I'll hold off on merging moving everything to Inline nodes until this is updated and merged. Lmk if you want me to update PR as I broke it.

Reply:
I've gone ahead and made some changes to inline the nodes for C. Luckily, C uses the same TS objects as CPP so there wasn't a lot of work required. :)